README.md

```diff
@@ -90,6 +90,7 @@ Note that in order to read the step outputs the action step must have an id.
         id: cpr
         uses: peter-evans/create-pull-request@v3
       - name: Check outputs
+        if: ${{ steps.cpr.outputs.pull-request-url }}
```

Comment:
I would prefer to use `pull-request-number` here for consistency.
```suggestion
        if: ${{ steps.cpr.outputs.pull-request-number }}
```

Reply:
Okay, why not :smiley: :+1:

Reply:
Done, pushed.